lib/linter.coffee

```diff
@@ -254,7 +260,12 @@ class Linter
   #        (optional)
   #   colStart: column to on which to start a higlight (optional)
   #   colEnd: column to end highlight (optional)
+  #   file: the file that a message was caused by (optional)
   computeRange: (match) ->
+    if match.file?
+      if match.file isnt @lintingFile
+        # TODO: show a message to indicate there are errors in other files
```

Comment:
@hd-deman Perhaps. I know little about Linter itself. Maybe it could be improved in another PR later?